fractions/jaeger/src/main/java/org/wildfly/swarm/jaeger/runtime/JaegerInstaller.java

```diff
@@ -0,0 +1,68 @@
+package org.wildfly.swarm.jaeger.runtime;
+
+import org.jboss.logging.Logger;
+import org.jboss.shrinkwrap.api.Archive;
+import org.wildfly.swarm.jaeger.JaegerFraction;
+import org.wildfly.swarm.spi.api.DeploymentProcessor;
+import org.wildfly.swarm.spi.runtime.annotations.DeploymentScoped;
+import org.wildfly.swarm.undertow.WARArchive;
+import org.wildfly.swarm.undertow.descriptors.WebXmlAsset;
+
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+
+import static com.uber.jaeger.Configuration.*;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+@DeploymentScoped
+public class JaegerInstaller implements DeploymentProcessor {
+    private static final Logger logger = Logger.getLogger(JaegerInstaller.class);
+
+    private final Archive<?> archive;
+
+    @Inject
+    private Instance<JaegerFraction> jaegerFractionInstance;
+
+    @Inject
+    public JaegerInstaller(Archive archive) {
+        this.archive = archive;
+    }
+
+    @Override
+    public void process() throws Exception {
+        JaegerFraction fraction = jaegerFractionInstance.get();
+        logger.info("Determining whether to install Jaeger integration or not.");
+        logger.info("JaegerFraction instance: " + fraction);
+
+        if (archive.getName().endsWith(".war")) {
+            logger.logf(Logger.Level.INFO, "Installing the Jaeger integration for the deployment %s", archive.getName());
+            WARArchive webArchive = archive.as(WARArchive.class);
+            WebXmlAsset webXml = webArchive.findWebXmlAsset();
+
+            logger.logf(Logger.Level.INFO, "Adding the listener org.wildfly.swarm.jaeger.deployment.JaegerInitializer");
+            webXml.addListener("org.wildfly.swarm.jaeger.deployment.JaegerInitializer");
+
+            setContextParamIfNotNull(webXml, JAEGER_SERVICE_NAME, fraction.getServiceName());
```

Comment:
Right, we are doing this because we want a "Swarm" native approach as well. The order should be: Swarm configuration (wherever it comes from), System property, Environment variables. The sysprop and env vars are the same as the "pure" Jaeger client.